.travis.yml

```diff
@@ -1,21 +1,17 @@
- branches:
-  only:
-    - module-7
-
 services:
   - docker
 
 script:
-  - echo "hello world"
   - docker build --target test --tag my-test-image . 
   - docker run my-test-image tests
   - docker run --env SECRET_KEY --env TRELLO_TODOID --env TRELLO_DONEID --env TRELLO_BOARDID --env TRELLO_KEY --env TRELLO_TOKEN my-test-image test_e2e
 
 env:
   global:
     - SECRET_KEY=secret-key
+    - secure: "wV+sfKT3Nsz2ll3GKf8SDwrbwTHvgTbz/9OHcQalaxW6tOS2LdX1GcHifTlK1JgNnEcORkvjk9asjcw3B1H9NbSZdw0+8pQn5//she0bALHIYDCYcZuRHqwjHtw3Y41YBbFE/8God/e6U/IwYy5Hrlt/SZavCmK/aw8ivfwehiZAe8oPhDq6KnJSErMgmdd2gS+xvClQuhK0p4isRel9XXIuARm1m8viAZtixQdbWH8lZ2pNEzQNdsLmJJ2WazjqkYz34g/sXYJcs8GIk5PlPj3HUWtRbX3DuikzmQdReuxkD7Q/3ONymNDb9xXd7jZGgCeMVGHd+A7Tu8B7eT9Vi7sUm9egs3NPTDD0tFoiKC8DR++WKMxuE8VaEMOnI/CfZaxOnqJOshCViCtOhxV6Il2KGtNjqjOZuf1IlUynD3yvI7V0trzyxwy728H8lCVSIfv7+dZ2Z5k3GX258ny6C00iD3zrqJ/OTv4AYPK3YEiyW/nRFhuDzrPI5JozQM8ibcvCrbYGf7qi6UWZxZ5X2BT1+KIk2ZVRUnSYJRbLeIsmy2MT2wiAcdx4itlrPwWmJiBRN4U8nhqjz/AUsWe+0cHzR1CAfsyjMeQOIEm70HAsU/dFaazHHCgaGFGy/2LyjH/de4SxjSvnF10cvH/7baOMD9aRSY4YeMIs3yNVLdE="
```

Comment:
Possibly unclear on what I said before, but no harm - I was just advising that if you ever need to figure out what variable this represents in future you can refer to the order Travis spits them out, not that there's a particular order in which you should list them